docs/sources/developers/plugins/cross-plugin-linking.md

```diff
@@ -0,0 +1,81 @@
+# Cross plugin links and commands
+
+Using the Plugin extensions API App, plugins can register placements of their own to display other plugins links. This cross-plugin linking creates a more immersive user experience for installed plugins.
+
+## How to create a placement within a plugin
+
+The `getPluginExtensions` method in `@grafana/runtime` creates a placement within your plugin. A placement is a way of specifying where in the plugin UI other plugins links or commands are rendered. `getPluginExtensions` takes an object consisting of the `placement` name, which must begin `plugin/<pluginId>`, and include any contextual information you would like to provide for the use of other plugins. It returns a list of `extensionLinks` and `extensionCommands` that your program can loop over.
+
+_Note: Creating a placement in a plugin creates a public interface for other plugins to interact with. Changes to the placement or the context it passes could break any plugin that attempts to register a link inside your plugin._
+
+```typescript
+import { getPluginExtensions } from '@grafana/runtime';
+import { isPluginExtensionLink, isPluginExtensionCommand } from '@grafana/data';
+import { LinkButton } from '@grafana/ui';
+
+function AppPlacementExample() {
+  const { extensions } = getPluginExtensions({
+    placement: 'plugin/another-app-plugin/menu', // Must start with "plugin/"
+    context: Object.freeze({
+      pluginId: 'another-app-plugin',
+    }),
```

Comment:
We can remove the `Object.freeze` here and do that internally before passing the context to the other plugins.